### DISCUSSION TITLE
Buying slots

### DISCUSSION THREAD

**Discussion by khoingt** [GriefPrevention/GriefPrevention#1549] [Q&A]
Just asking, can I make a GUI so that the player can buy more claim slots?
TY!

**Comment by Jikoo** [comment]
Yes, you absolutely could create an addon to do that. You're probably after `PlayerData#setAccruedClaimBlocks(Integer)` or `PlayerData#setBonusClaimBlocks(Integer)`.

If you're asking "can I create a GUI using an external GUI plugin" probably yes, `/adjustbonusclaimblocks` can be used, and you could attach a cost to the button assuming the GUI plugin supports that. That said, if you're having difficulty configuring something in another plugin, you should contact that plugin's maintainers for support.

**Reply by khoingt** [reply]
can you suggest me a plugin that does that?
if it's free then, great!

**Reply by Jikoo** [reply]
I cannot personally recommend one as I've never used any, but here's an admincraft thread with 4 options (3 free, 1 premium) you could look into: https://www.reddit.com/r/admincraft/comments/gwm5ox/best_custom_gui_plugin/
